edx_proctoring/templates/practice_exam/entrance.html

```diff
@@ -74,10 +74,10 @@ <h3>
         ).fail(function(){
           enableClickEvent();
           var msg = gettext(
-            "There has been a problem starting your exam.\n\n" +
-            "Possible reasons are that your account has not been fully activated,\n" +
-            "you have are experiencing a network connection problem, or there has been\n" +
-            "a service disruption. Please check these and try again."
+            "There is a network problem or service connection issue.\n\n " +
```

Comment:
The message update here removes the scenario where `service disruption` is a possible. Can we add that back in?
In the event of service disruption, like what we are experiencing with Proctortrack right now, showing this message makes our learners think there is something wrong with their environment.

Reply:
Thanks for catching this, I had copied what was in the ticket but I agree that this wording should be kept in.